### ISSUE TITLE
rebuild parts bulkloader

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#2603]
It's trying to do too much stuff, its complicated to use, complicated to debug, difficult to maintain, and nobody (especially me!) quite knows what all it does. 

Create as many simple tools as necessary to maintain functionality.

Refs:

https://github.com/ArctosDB/arctos/issues/1526
https://github.com/ArctosDB/arctos/issues/1792
https://github.com/ArctosDB/arctos/issues/2433

**Comment by Jegelewicz** [comment]
We also need to consider creating a similar user experience for all of the separate bulkloading tools.

I recently tried to use the part bulkload tool to load 3,210 parts. The tool timed out, but everything remains in the table, including stuff that loaded. There is no "delete all loaded", so it is either check thousands of boxes and "delete checked" (which doesn't work - I got a message something like "this only works for fewer than 1,000 items") or download a csv, delete the loaded stuff plus some extraneous columns, delete everything from the tool, then upload the revised file. We need to create a consistent experience for all of the these tools and let users know about how much each of them can handle.

**Comment by Jegelewicz** [comment]
BTW - one suggestion for simplification. Tools should either add new stuff, edit existing stuff, or remove stuff not attempt to do more than one of those things.

**Comment by dustymc** [comment]
That's the core of this issue, but I think we should go even further by removing "modify." That's essentially what lead us here, and I've regretted it every time I've had to do anything with that form. The very minor benefits (don't have to copy the identifiers column into a second spreadsheet, essentially) are drastically outweighed by the costs of dealing with the mess, for both users and developers. All(ish) 'modify' operations can be done with delete and create, and we should use that pathway when possible.
